### PR TITLE
Docs + Feature: Update README.md and Add Synchronous Badge to Online Courses

### PR DESCRIPTION
**Pull request by JoshoTheMosho**
# Description

Added steps to fix compilation errors on windows environments regarding line break changes that prevents the application from compiling.

## General Comments

Let me know if this happens to break everything

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ If you're interested developing this application, follow these steps to get runn
 
 You will now be able to develop the frontend React application. Any changes made to the code will hot-reload upon save.
 
+Failing to compile on a windows environment?
+Run the following commands and then retry step 4:
+`git fetch --all`
+`git reset --hard`
+
```
Comment (FlyteWizard) [inline README.md:34]: Note: If the above is failing to compile on a Windows machine, you will first need to run `git fetch --all` and then `git reset --hard`. After successfully running these two commands, you can then retry step 4 (`npm start`).

**Comment by keithradford** [comment]
Throwing it out there for next time: you can clone the main repo now since you're a part of the CourseUp team, don't have to fork!

**Comment by aomi** [comment]
It would be good to separate this into two PRs (document and feature) one. I know these changes are trivial but it would be a good exercise to do it here so you're familiar with the process. 
Like Keith mentioned, you don't have to work off a fork as you have write access to the repository.
To make another PR, I would suggest the following.
```bash
git checkout main # to get back to main
git checkout -b josho/add-windows-note # just an example name
git cherry-pick 4d2080ab1ef85b1b2f573ee36fc1453bb7de2d54 # this is the SHA of the commit that edits the README
git push origin ... # truncated
```
From there, you can either `git revert 4d2080ab1ef85b1b2f573ee36fc1453bb7de2d54` and keep this branch and make it your "feature" branch or you can make another new branch and do the same thing by cherry-picking commits. Happy to walk you through this if you get stuck as it's a super useful skill.

**Comment by keithradford** [comment]
Closing this since separate PRs have been made for each feature :)

**Pull request closed without merging**